Tests/SwiftProtobufTests/Test_Naming.swift

```diff
@@ -184,3 +184,165 @@ class Test_FieldNamingInitials: XCTestCase {
     msg.clearTheURL()
   }
 }
+
+class Test_ExtensionNamingInitials_MessageScoped: XCTestCase {
+  func testLowers() {
+    var msg = SwiftUnittest_Names_ExtensionNamingInitials()
+
+    msg.SwiftUnittest_Names_Lowers_http = 1
```

Comment:
I like the way you have it here, personally, with simple combining of the scope name and field name.